hole/hole.go

```diff
@@ -116,15 +116,17 @@ func Play(ctx context.Context, holeID, langID, code string) (score Scorecard) {
 		}
 	}
 
+	const maxLength = 1024 * 1024
+
 	// Trim trailing whitespace.
-	score.Stderr = bytes.TrimRightFunc(stderr.Bytes(), unicode.IsSpace)
+	score.Stderr = bytes.TrimRightFunc(stderr.Next(maxLength), unicode.IsSpace)
```

Comment:
Neat, I didn't know about `Next`, I should read the buffer docs more, this is a really elegant solution to the problem.